app/controllers/locations_controller.rb

```diff
@@ -35,4 +38,15 @@ def show
   def locations
     Locations.nearest_to_postcode(params[:postcode], limit: 5)
   end
+
+  def set_breadcrumbs_for_search
+    breadcrumb Breadcrumb.book_an_appointment
+    breadcrumb Breadcrumb.how_to_book
+  end
+
+  def set_breadcrumbs_for_location
+    breadcrumb Breadcrumb.book_an_appointment
+    breadcrumb Breadcrumb.how_to_book
+    breadcrumb Breadcrumb.find_a_location
```

Comment:
Ah, my misunderstanding. We can get rid of `Breadcrumb.find_a_location` completely then which will simplify things.